src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -96,6 +97,16 @@ export const TradeConfirm = ({ history }: RouterProps) => {
       .toNumber(),
   )
 
+  const gasFeeTradeAmountPercentage = bnOrZero(fees?.fee)
+    .times(fiatRate)
+    .dividedBy(
+      bnOrZero(fromBaseUnit(bnOrZero(trade?.sellAmount), trade?.sellAsset.precision ?? 0)).times(
+        bnOrZero(sellAssetFiatRate),
+      ),
+    )
+    .times(100)
+    .toNumber()
+
```

Comment:
Stylistic change for readability.

```suggestion
  const feeFiatValue = bnOrZero(fees?.fee).times(fiatRate)
  const tradeFiatValue = bnOrZero(fromBaseUnit(bnOrZero(trade?.sellAmount), trade?.sellAsset.precision ?? 0))
    .times(bnOrZero(sellAssetFiatRate))
  const gasFeeTradeAmountPercentage = feeFiatValue.dividedBy(tradeFiatValue).times(100).toNumber()
```